ordered logistic regression.ipynb

```diff
@@ -16,34 +16,50 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">&nbsp;2) r_hat &lt;&lt; 1
</pre><p>isn't the red flag when it's greater than 1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MarcoGorelli/pydataglobal-21/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Ok. Sorry. I hadn't noticed back when I did the PR.